docs/src/pages/components/data-grid/rows/rows.md

```diff
@@ -53,6 +53,15 @@ Alternatively, if you would like to delete a row, you would need to pass an extr
 apiRef.current.updateRows([{ id: 1, _action: 'delete' }]);
 ```
 
+### Throttling grid update [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-pro/)
+
+If you have high frequency updates, but you want to keep good performances, you can pass set the `throttleRowsMs` prop to define the maximum update frequency.
```

Comment:
```suggestion
Whenever the rows are updated, the grid has to apply the sorting and filters. This can be a problem if you have high frequency updates. To maintain a good performance, the grid allows to batch the updates and only apply them after a period of time. The `throttleRowsMs` prop can be used to define the frequency (in milliseconds) at which rows are updated.
```